sql/parser/all_keywords.awk

```diff
@@ -6,7 +6,7 @@ BEGIN {
     print "var keywords = map[string]int{"
 
     # This variable will be associated with a pipe for intermediate output.
-    sort = "sort"
+    sort = "sort -t \"\\\"\" -k 2,2"
```

Comment:
I would prefer to use `LC_ALL=C` to remove the influence of the environment locale. I'm not sure where the best place to set this is, but `sort = "env LC_ALL=C sort"` should work.